### PR TITLE
Resolves #1026

### LINKED ISSUE #1026: Clarify the acceptable values for -GptType
I haven't done exhaustive tests, but it appears that not all GUID strings are acceptable values.

See https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netcore-2.2
VALID
* Guid.ToString("B")

NOT VALID ("The specified partition type is not valid." error)
* Guid.ToString("N")
* Guid.ToString("D")
* Guid.ToString("P")
* Guid.ToString("X")

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8e854ce1-0e6c-2261-c35d-94e2faaa96bc
* Version Independent ID: 95312c3e-0f34-cb59-04f1-3cbe6762287a
* Content: [New-Partition (storage)](https://docs.microsoft.com/en-us/powershell/module/storage/new-partition?view=win10-ps)
* Content Source: [docset/windows/storage/new-partition.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/storage/new-partition.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

### PR DESCRIPTION
**Pull request by velkovb**
Changed examples for GUID types to use braces instead of parentheses.
Added a line to specify the format explicitly.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1026

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor copy editing is needed for this PR. Thanks.

**Comment by tseward** [comment]
I'm OK with the PR.

**Comment by dariomws306** [comment]
I' am agree with Trevor!

**Comment by get-itips** [comment]
Ok!

File: docset/windows/storage/new-partition.md [hunk 1]
```diff
@@ -211,18 +211,20 @@ Accept wildcard characters: False
 ```
 
 ### -GptType
-Specifies the type of GPT partition to create (by GUID).
+Specifies the type of GPT partition to create (by GUID). 
+The format should be 32 digits separated by hyphens, enclosed in braces:
```
Comment (JohanFreelancer9) [inline docset/windows/storage/new-partition.md]:
```suggestion
The format should be 32 digits, separated by hyphens, enclosed in braces:
```

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged.

**Pull request merged**